modules/nf-core/segemehl/align/main.nf

```diff
@@ -0,0 +1,54 @@
+process SEGEMEHL_ALIGN {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda (params.enable_conda ? "bioconda::segemehl=0.3.4" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/segemehl:0.3.4--hc2ea5fd_5':
+        'quay.io/biocontainers/segemehl:0.3.4--hc2ea5fd_5' }"
+
+    input:
+    tuple val(meta), path(reads)
+    path(fasta)
+    path(index)
+
+    output:
+    tuple val(meta), path("${meta.id}/*"), emit: results
```

Comment:
Shouldn't this be prefix and not meta.id?

Reply:
Just tested it locally, 

`PROFILE=docker pytest --tag segemehl/align --symlink --keep-workflow-wd`

 - with `${prefix}`: failed
 - with `${meta.id}`: passed

🤔

Reply:
```suggestion
     tuple val(meta), path("${prefix}/*"), emit: results
```
Sorry that I wasn't being very clear before. Because of the conditional you have in line 24, if a user defines his/her own prefix, then the prefix variable in the script will be set to that user-defined value and not meta.id. So when you have an user-defined prefix, if you are looking for a pattern that has the value of meta.id in it, you would miss capturing the output. But using `path("${prefix}/*")` should cover all scenarios.